authboss.go

```diff
@@ -89,14 +98,17 @@ func (a *Authboss) UpdatePassword(ctx context.Context, user AuthableUser, newPas
 // Returns nil on success otherwise there will be an error. Simply a helper
 // to do the bcrypt comparison.
 func VerifyPassword(user AuthableUser, password string) error {
+	// TODO: function can be used ONLY if no custom hasher was configured in global ab.config
```

Comment:
This comment belongs at the top-level function. We should also introduce a new function that hangs off of `Authboss` that replaces this one.